third-party/realdds/include/realdds/dds-exceptions.h

```diff
@@ -14,8 +12,11 @@
 #define DDS_THROW( ERR_TYPE, WHAT )                                                                                    \
     do                                                                                                                 \
     {                                                                                                                  \
-        LOG_ERROR( "throwing: " << WHAT );                                                                             \
-        throw realdds::dds_##ERR_TYPE( rsutils::string::from() << WHAT );                                              \
+        std::ostringstream os__;                                                                                       \
```

Comment:
Why double underscores?

Reply:
So it won't interfere with any other possible local variables